### ISSUE TITLE
fu-i2c-device: PS175 and RTD2142 devices do not appear in get-devices after incorporate change.

### ISSUE THREAD

**Issue by bleungatchromium** [fwupd/fwupd#5088]
**Describe the bug**

Ever since commit a694a1d915efe714bd848add5983cf9bc589c513, Chromeboxes with PS175 or RTD2142 onboard devices have stopped detecting 

**Steps to Reproduce**
Steps to reproduce the behavior.
Deploy 1.8.2 or later on chromebox with i2c devices (PS175 or RTD2142).

**Expected behavior**
device is properly found. Example:
```
# fwupdmgr get-devices
Noibat
?
??DA4064:
?     Device ID:          eafa35acc9a754da25475e870cca0b4460daa4ed
?     Current version:    0x3535303430393139
?     Vendor:             EMMC:0x000045
?     Serial Number:      0x86a60316
?     GUIDs:              2212d0ea-5112-5ef4-8b36-59756743e25d ? EMMC\NAME_DA4064
?                         4311de42-bed8-5bbc-b776-fffd6fe8d5b6 ? EMMC\NAME_DA4064&REV_0x3535303430393139
?                         5a02344d-585b-507b-ad59-19e96c03ca46 ? EMMC\MAN_0045&OEM_0100
?                         1d13840a-391c-5f1e-9c98-21814fbb0503 ? EMMC\MAN_0045&OEM_0100&NAME_DA4064
?                         90660737-97f9-5fe7-9f1d-af69b8b8d8f9 ? EMMC\MAN_0045&NAME_DA4064&REV_0x3535303430393139
?                         4a018b5a-bb17-5d43-bea8-1784ee947bb7 ? EMMC\MAN_0045&OEM_0100&NAME_DA4064&REV_0x3535303430393139
?     Device Flags:       ? Internal device
?                         ? Updatable
?   
??PS175:
      Device ID:          e6cbb94f70a759bae192861b67c828457d66210f
      Current version:    9.69
      Vendor:             Parade Technologies (PCI:0x1AF8, OUI:001CF8)
      GUIDs:              c46df45b-16f2-5eb2-be8d-66f5af40cca8 ? PARADE-LSPCON\NAME_1AF80175:00
                          c146ccc9-58b6-517c-97f6-9c55a0bd39d3 ? I2C\NAME_1AF80175:00
      Device Flags:       ? Internal device
                          ? Updatable
                          ? Cryptographic hash verification is available
                          ? Device stages updates
                          ? Unsigned Payload
```

**fwupd version information**
Please provide the version of the daemon and client.

```
# fwupdmgr --version
runtime   org.freedesktop.fwupd         1.8.2
compile   org.freedesktop.gusb          0.3.10
runtime   org.kernel                    4.19.256-13656-gcba11f2b8ff4
compile   com.hughsie.libjcat           0.1.8
compile   org.freedesktop.fwupd         1.8.2
runtime   org.freedesktop.gusb          0.3.10
```

Please note how you installed it (`apt`, `dnf`, `pacman`, source, etc):

**fwupd device information**
Please provide the output of the fwupd devices recognized in your system.

```
localhost ~ # fwupdmgr get-devices --show-all-devices
Noibat
?
??DA4064:
?     Device ID:          eafa35acc9a754da25475e870cca0b4460daa4ed
?     Current version:    0x3535303430393139
?     Vendor:             EMMC:0x000045
?     Serial Number:      0x86a60316
?     GUIDs:              2212d0ea-5112-5ef4-8b36-59756743e25d ? EMMC\NAME_DA4064
?                         4311de42-bed8-5bbc-b776-fffd6fe8d5b6 ? EMMC\NAME_DA4064&REV_0x3535303430393139
?                         5a02344d-585b-507b-ad59-19e96c03ca46 ? EMMC\MAN_0045&OEM_0100
?                         1d13840a-391c-5f1e-9c98-21814fbb0503 ? EMMC\MAN_0045&OEM_0100&NAME_DA4064
?                         90660737-97f9-5fe7-9f1d-af69b8b8d8f9 ? EMMC\MAN_0045&NAME_DA4064&REV_0x3535303430393139
?                         4a018b5a-bb17-5d43-bea8-1784ee947bb7 ? EMMC\MAN_0045&OEM_0100&NAME_DA4064&REV_0x3535303430393139
?     Device Flags:       ? Internal device
?                         ? Updatable
?   
??Display controller:
      Device ID:          5792b48846ce271fab11c4a545f7a3df0d36e00a
      Current version:    02
      Vendor:             PCI:0x8086
      GUIDs:              5589d313-88e5-54d3-8f4c-b2ee3882d4bc ? PCI\VEN_8086&DEV_9B41
                          9779d679-0f31-5dd5-b3c2-f6ff275a9ca9 ? PCI\VEN_8086&DEV_9B41&REV_02
                          2ffa2655-4d89-5462-81f9-8db939585b5e ? PCI\VEN_8086&DEV_9B41&SUBSYS_80862212
                          e4ef1dc4-be60-5ebc-99b7-0c2b8475c0eb ? PCI\VEN_8086&DEV_9B41&SUBSYS_80862212&REV_02
                          25be981a-c8a2-5d3a-82cc-39c2a795effb ? PCI\VEN_8086&DEV_9B41&REV_00
                          d5a11444-73b3-5589-b182-de5c6cd56e3c ? PCI\VEN_8086&DEV_9B41&SUBSYS_80862212&REV_00
      Device Flags:       ? Internal device
                          ? Cryptographic hash verification is available
```

**Additional questions**

- Operating system and version: ChromeOS
- Have you tried rebooting? Yep
- Is this a regression? Yep

**Comment by bleungatchromium** [comment]
WARNINGS from fwupd are here:
https://gist.githubusercontent.com/bleungatchromium/8f1ee25dd765a49640a8f15b1a41d957/raw/46a45f2a0e115d9d5bf42adfab053a4546903bff/gistfile1.txt

**Comment by bleungatchromium** [comment]
@djcampello as FYI.

**Comment by bleungatchromium** [comment]
As discussed with @hughsie this morning, he and I will work on this tomorrow and nail it down.

We also need to add these to device-tests so we don't regress this in the future.

**Comment by bleungatchromium** [comment]
Thanks, Richard. Fixed in 1.8.6 with #5132
